server/application-server/src/main/java/de/tum/in/www1/hephaestus/codereview/actor/Actor.java

```diff
@@ -0,0 +1,92 @@
+package de.tum.in.www1.hephaestus.codereview.actor;
+
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+import de.tum.in.www1.hephaestus.codereview.comment.Comment;
+import de.tum.in.www1.hephaestus.codereview.pullrequest.Pullrequest;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.OneToMany;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Entity
+@Table(name = "actors")
+@Getter
+@Setter
+@NoArgsConstructor
+@JsonInclude(JsonInclude.Include.NON_EMPTY)
+public class Actor {
```

Comment:
Agreed, the naming comes from Github's GraphQL API, which also calls it `Actor`:
![image](https://github.com/user-attachments/assets/c3d8d5dc-8e4a-4122-9348-fd3f6942ced1)

I would be fine with renaming it. Do we plan on using different tables for Github and Gitlab? Then `GHUser` and `GLUser` would be perfect. Otherwise maybe `ExternalUser`?

Reply:
I think we will have one table for each entity and try our best to union it